fleet-state/fleet-state.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+NAMESPACE=$1
+
+if [ -z $NAMESPACE ];
+then
+    echo "Usage: ./fleet-state.sh <namespace>"
+    exit 1
+fi
+
+kubectl get namespace $NAMESPACE 1>/dev/null 2>&1
+if [ $? -ne 0 ];
+then
+    echo "Namespace ${NAMESPACE} does not exist"
+    exit $?
+fi
+
+${DIR}/nomad/nomad.sh $NAMESPACE
+${DIR}/vm-service/vms.sh $NAMESPACE
+# ${DIR}/distributor/distributor.sh $NAMESPACE
```

Comment:
Is it correct that this is commented out?

Reply:
Yes